### DISCUSSION TITLE
Strategies with 90%+ winrate

### DISCUSSION THREAD

**Discussion by mujahidazamcsm** [conor19w/Binance-Futures-Trading-Bot#16] [Ideas]
Hi @conor19w 

On youtube we have several strategies with 90%+ winrate - some even claiming 100% winrate. Do u have the time to implement any of those?

**Comment by conor19w** [comment]
Yeah I've got time I can add to the to-do list.
Inflated results usually and most channels don't take into account trading fees.
Also most only do 100 trades backtest.
Most of the strategies here have been taken from such YouTube channels.

**Reply by mujahidazamcsm** [reply]
Cool. Do u have a list which u plan to implement? or consider some suggestions also.

**Reply by conor19w** [reply]
Just open to suggestions, but I have a list of suggestions on the readme page that I'll add to
